src/Numerics/ODESolvers/MultirateInfinitesimalStepMethod.jl

```diff
@@ -151,6 +190,48 @@ mutable struct MultirateInfinitesimalStep{
     end
 end
 
+function MultirateInfinitesimalStep(
+    method::Symbol,
```

Comment:
can pass the object instead of the symbol
```suggestion
    mis, 
```